codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/rustlang/RustWriter.kt

```diff
@@ -468,7 +468,7 @@ class RustWriter private constructor(
                     debugMode = debugMode,
                     devDependenciesOnly = true,
                 )
-
+                fileName == "package.json" -> rawWriter(fileName, debugMode = debugMode)
```

Comment:
Ideally we should move this to a specific rust writer, but the amount of refactor to do it did not seems worth it.